UPGRADE.md

```diff
@@ -8,6 +8,11 @@ awareness about deprecated code.
 
 # Upgrade to 4.0
 
+## BC BREAK: Removed lock-related `AbstractPlatform` methods
+
+The methods `AbstractPlatform::getReadLockSQL()`, `::getWriteLockSQL()` and `::getForUpdateSQL` have been removed
```

Comment:
```suggestion
The methods `AbstractPlatform::getReadLockSQL()`, `::getWriteLockSQL()` and `::getForUpdateSQL()` have been removed
```

Reply:
Good catch, I've also pushed c80d4cb5836a6c3e3c72a09ea720ccd88785e325 to 3.8.x.